bundles/org.openhab.binding.mail/src/main/java/org/openhab/binding/mail/internal/POP3IMAPHandler.java

```diff
@@ -107,9 +107,12 @@ public void dispose() {
     }
 
     private void refresh() {
-        Session session = Session.getDefaultInstance(new Properties(), null);
+        Properties props = new Properties();
+        props.setProperty("mail.imap.starttls.enable", config.starttls.toString());
```

Comment:
Always set to true, use imap or pop3 depending on the selected protocol (starttls is also available for pop3)